universal-login-wallet/src/ui/react/Modals/Transfer/ModalTransfer.tsx

```diff
@@ -10,7 +10,10 @@ const ModalTransfer = () => {
   const [modal, setModal] = useState('transferAmount');
 
   const {walletService, sdk} = useServices();
-  const [transferDetails, setTransferDetails] = useState({transferToken: ETHER_NATIVE_TOKEN.address} as TransferDetails);
+  // FIXME: This cast looks like a bug
+  const [transferDetails, setTransferDetails] = useState(
```

Comment:
It looks like the default value of `transferDetails`. We don't want to have a default amount, but the default token is ok. I don't see a bug here. We can remove this comment and add a task to look at this. Maybe we can move this default value to the bottom? Let's discuss later.